website/content/docs/known-issues-and-solutions.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Known Issues and Solutions
+weight: 20
+group: support
+---
+
+## Background
+
+There are a couple of issues that users face while integrating or working with Netlify CMS and while the community is doing their best fixing them, here is a list of known issues and their solutions.
+
+## API Errors
+
+A list of error messages and their possible cause(s) to provide more context while debugging.
+
+#### 1. Error Message.
+
+```
+API_ERROR: {"message":"404 File Not Found"}
+```
+
+#### Possible Causes.
+
+- The collection folder or file property is missing a value (a dedicated file or folder, can't be blank or root). The file or folder should be relative to root (so `src/posts` for example).
```

Comment:
I think these scenarios should be fixed (we don't throw an error, but just won't list any entries).
Do you mind verifying it?

Reply:
I will verify this.

Reply:
I think you are right.

Reply:
No, it doesn't throw an error.